frontend/src/apiServices/service.ts

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import {IngredientModel} from "../components/Model";
+
+export function getGinDrinks() {
+    return axios.get("https://thecocktaildb.com/api/json/v1/1/filter.php?i=Vodka")
+        .then(response => response.data)
```

Comment:
Here and in the other methods I would type the `response` variable, so that typscript know the type of data. In this case `response` would be `(response: AxiosResponse<Cocktail[]>)`.